tools/frrcommon.sh.in

```diff
@@ -208,7 +208,7 @@ daemon_stop() {
 
 	[ -r "$pidfile" ] || fail="pid file not found"
 	$all && [ -n "$fail" ] && return 0
-	[ -z "$fail" ] && pid="$(cat \"$pidfile\")"
```

Comment:
90% sure only the backslashes should be removed

Reply:
why not 100%? :smile: Fixed.